GOVERNANCE.md

```diff
@@ -68,8 +68,8 @@ The current Stakeholders of the MaterialX TSC are:
 - Rafal Jaroszkiewicz - SideFX
 - Lee Kerley - Sony Pictures Imageworks
```

Comment:
As an aside, Lee is no longer at SPI, so his affiliation should be corrected.

And we should appoint a new stakeholder, I guess.

Reply:
Good point, and let's chat offline about appointing a new representative from SPI.